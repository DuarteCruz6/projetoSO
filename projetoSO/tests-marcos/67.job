WRITE [(i8,1)(q0,79)(l7,72)]
READ [l7]
WRITE [(p3,48)(b5,58)(v8,29)]
WRITE [(z1,81)(w6,93)(z3,52)]
BACKUP
BACKUP
WRITE [(e3,75)(y2,61)(x0,63)]
BACKUP
DELETE [z3,x0,q0,l7,i8]
WRITE [(e7,30)(p0,52)(s9,43)(k0,74)]
WRITE [(p9,78)(d8,85)]
WRITE [(b9,54)(o6,38)(f5,13)(k3,35)(q7,2)]
WRITE [(g1,6)(a8,90)(q0,68)]
WRITE [(x9,18)(l8,70)]
READ [d8]
BACKUP
WRITE [(g9,69)(x0,42)(g7,93)(m9,34)(r3,100)]
BACKUP
WRITE [(n2,94)(s3,87)(f0,28)]
READ [k0]
READ [k3]
WRITE [(i2,20)(h7,16)(t0,55)(i8,58)(g0,21)]
WRITE [(u7,66)(f4,51)(u9,78)]
WRITE [(q3,28)(n3,70)(d5,3)(i0,49)]
DELETE [n3]
WRITE [(f3,75)]
DELETE [g9,q3,p0,i8,s3]
WAIT 2000
READ [s9]
WRITE [(c3,50)]
WRITE [(a1,60)(t3,88)(n7,75)(a7,58)(k7,77)]
DELETE [g0,p9]
WRITE [(u0,86)(o8,5)(j1,60)]
WRITE [(b3,42)(p7,54)(b4,50)(o2,27)]
DELETE [g7,b9]
WRITE [(l0,95)(u2,97)(e6,39)(a4,72)(u8,36)]
DELETE [d8,q0,e6,k0]
DELETE [m9,r3,x0,t3,a1]
READ [f0]
WRITE [(b7,10)(m9,7)(a2,81)(a1,37)]
WRITE [(t4,65)(r5,42)]
WRITE [(n1,75)(v0,62)(z7,14)(p2,40)(p1,63)]
WRITE [(x8,81)(x8,42)(b8,28)(v4,34)]
DELETE [b4,y2]
WRITE [(y7,55)(t6,99)(z3,55)]
DELETE [m9,k7,p7,f0]
WAIT 2000
WRITE [(i9,19)(c1,98)(l3,54)(d9,1)]
WRITE [(k8,15)(m5,39)(i4,6)(b0,79)]
WRITE [(s6,27)(v9,90)]
WRITE [(j5,14)(i7,9)]
WRITE [(s8,11)(r1,17)(h2,62)(x7,50)]
READ [z3]
READ [i2]
WRITE [(j0,18)]
WRITE [(n0,53)(j4,54)(k5,74)(c2,100)]
BACKUP
WRITE [(u6,36)(u6,93)]
WRITE [(h3,78)(l7,18)(y5,33)(v3,88)]
WRITE [(d8,15)(x1,56)(p5,87)(m9,11)(i6,52)]
WAIT 3000
