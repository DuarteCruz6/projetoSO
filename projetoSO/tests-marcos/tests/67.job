WRITE [(w3,46)(a7,49)(t0,97)]
DELETE [a7,w3]
WRITE [(a4,6)(t4,35)(e6,47)]
DELETE [e6,a4]
READ [t4]
READ [t0]
READ [t4]
WRITE [(p8,92)(z2,66)]
READ [z2]
WAIT 1000
DELETE [z2,p8,t4,t0]
WRITE [(d5,48)(c7,97)(t7,9)(j7,99)]
READ [c7]
WAIT 1000
DELETE [j7,c7]
WRITE [(m9,72)(i0,77)(h2,91)(t5,79)]
WRITE [(h9,45)(q1,39)(i5,15)]
WRITE [(y2,66)(w2,17)]
WRITE [(b1,61)(f0,76)(u1,18)(t6,47)(v8,54)]
WRITE [(e2,73)(e9,1)]
WRITE [(g6,88)(i8,14)]
WRITE [(m1,24)(c9,62)(x3,53)(m2,65)(m7,98)]
WAIT 3000
SHOW
DELETE [y2,i5]
DELETE [m9,v8,q1,x3]
READ [m7]
WRITE [(n0,10)(x4,57)]
WAIT 3000
