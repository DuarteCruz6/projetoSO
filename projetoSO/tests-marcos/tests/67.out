[(t4,35)]
[(t0,97)]
[(t4,35)]
[(z2,66)]
[(c7,97)]
(b1, 61)
(c9, 62)
(d5, 48)
(e9, 1)
(e2, 73)
(f0, 76)
(g6, 88)
(h9, 45)
(h2, 91)
(i8, 14)
(i5, 15)
(i0, 77)
(m7, 98)
(m2, 65)
(m1, 24)
(m9, 72)
(q1, 39)
(t6, 47)
(t5, 79)
(t7, 9)
(u1, 18)
(v8, 54)
(w2, 17)
(x3, 53)
(y2, 66)
[(m7,98)]
